Gulpfile.js

```diff
@@ -138,22 +187,23 @@ function webpackBuild() {
       net: "empty",
     },
     output: {
-      filename: "babel.js",
-      library: "Babel",
+      filename: opts.filename,
+      library: opts.library,
       libraryTarget: "umd",
     },
     plugins: [
       new webpack.DefinePlugin({
         "process.env.NODE_ENV": '"production"',
-        BABEL_VERSION: JSON.stringify(version),
+        "process.env": JSON.stringify({ NODE_ENV: "production" }),
```

Comment:
is this the same ^?

Reply:
Yeah why do you need this? NODE_ENV is already defined two lines above.

Reply:
It doesn't work for me with cases like `var env = process.env`: https://github.com/ai/browserslist/blob/master/index.js#L35